services/rds/src/main/java/org/finra/gatekeeper/configuration/GatekeeperOverrideProperties.java

```diff
@@ -51,16 +52,34 @@ public GatekeeperOverrideProperties setOverrides(Map<String, Map<String, Map<Str
         return overrides;
     }
 
-    public Map<String, Map<String, Integer>> getOverridePolicy(GatekeeperRdsRole role) {
-        return (Map<String, Map<String, Integer>>)this.overrides.get(role.toString().toLowerCase());
+    public Map<String, Map<String, Map<String, Integer>>> getOverridePolicy(Map<String, RoleMembership> roleMemberships, boolean isApprover) {
```

Comment:
I think we should treat the inner Map of Maps as a POJO, the three layers of map is kinda confusing